tests/model/test_model.py

```diff
@@ -2079,20 +2101,60 @@ def test_toggle_stream_pinned_status(
                     "id": 0,
                 },
                 True,
+                False,
+                False,
+                id="op_stop",
             ),
         ],
-        ids=["not_in_pm_narrow", "not_in_pm_narrow_with_sender", "start", "stop"],
     )
-    def test__handle_typing_event(self, mocker, model, narrow, event, called):
+    def test__handle_typing_event(
+        self,
+        mocker,
+        model,
+        narrow,
+        event,
+        is_animation_in_progress,
+        expected_is_recipient_typing,
+        expected__show_typing_animation,
+    ):
         event["type"] = "typing"
 
-        mocker.patch("zulipterminal.ui.View.set_footer_text")
+        mocker.patch("zulipterminal.model.Model._show_typing_animation")
         model.narrow = narrow
         model.user_dict = {"hamlet@zulip.com": {"full_name": "hamlet"}}
+        model.is_recipient_typing = False
+        model.is_typing_animation_in_progress = is_animation_in_progress
 
         model._handle_typing_event(event)
 
-        assert model.controller.view.set_footer_text.called == called
+        assert model.is_recipient_typing == expected_is_recipient_typing
+        assert model._show_typing_animation.called == expected__show_typing_animation
+
+    @pytest.mark.parametrize(
+        "event, is_set_footer_animation_called",
+        [
+            case({"op": "start"}, True, id="in_pm_narrow_with_sender_op_start"),
+            case({"op": "stop"}, False, id="op-stop"),
+        ],
+    )
+    def test__show_typing_animation(
+        self, mocker, model, event, is_set_footer_animation_called
+    ):
+        mocker.patch("zulipterminal.ui.View.set_footer_text")
+        recipient = {"full_name": "hamlet"}
+        model.is_recipient_typing = event["op"] == "start"
+        model.typing_event_start_time = datetime.now() - timedelta(seconds=14.5)
+
+        model._show_typing_animation(recipient)
+
+        if is_set_footer_animation_called:
+            assert model.controller.view.set_footer_text.called is True
```

Comment:
I understand the general benefit of erring towards being verbose, but I think this statement will maintain readability even without the `is True` part of it. We can just
```python
assert that something is called
```
instead of 
```python
assert that something is called is True
```
The other benefit is consistency with the `assert`s in the other tests.